ers/sql.go

```diff
@@ -121,6 +124,141 @@ func (rdr *SQLEventReader) Serve() (err error) {
 	return
 }
 
+// Creates mysql conditions used in WHERE statement out of filters
+func valueQry(ruleType, elem, field string, values []string, not bool) (conditions []string) {
```

Comment:
I would vote to move this function in utils so we can use it in the future for other cases (ie: EEs).